documentation/modules/auxiliary/server/socks4a.md

```diff
@@ -0,0 +1,60 @@
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: ```use auxiliary/server/socks4a```
```

Comment:
I thought it was a bit odd... I was trying to conform to the template https://raw.githubusercontent.com/rapid7/metasploit-framework/master/documentation/modules/module_doc_template.md in the off-chance there's some kind of automation generating funky docs off of these files. I'll swap to the singles :+1: